task/bq2bq/README.md

```diff
@@ -0,0 +1,25 @@
+# Bigquery SQL transformation
+
+Optimus plugin that supports variety of load methods to execute SQL transformations.
+Features
+- Automatic dependency resolution
+- Append to a partition
+- Replace table partition
+- Merge statements
+- BQ Scripts
+
+#### Load Method
+
+The way data loaded to destination table depends on the partition configuration of the destination tables
+
+| Load Method  | No Partition                                                                                   | Partitioned Table                                                                          |
+| -------------|------------------------------------------------------------------------------------------------| -------------------------------------------------------------------------------------------|
+| APPEND       | Append new records to destination table                                                        | Append new records to destination table per partition based on localised start_time        |
+| REPLACE      | Truncate/Clean the table before insert new records                                             | Clean records in destination partition before insert new record to new partition           |
+| REPLACE_MERGE| Doesn't work for non partitioned tables and partitioned tables with ingestion time             | Same as REPLACE but experimental                                                           |
```

Comment:
by reading this, not getting what experimental means for this and whether I should use REPLACE_MERGE instead of REPLACE

Reply:
done